includes/wikia/VariablesBase.php

```diff
@@ -2664,7 +2664,14 @@
  * @see extensions/wikia/FeedsAndPosts
  * @var bool $wgEnableFeedsAndPostsExt
  */
-$wgEnableFeedsAndPostsExt = false;
+$wgEnableFeedsAndPostsExt = true;
```

Comment:
does it mean that fepo will be enabled on all wikis by default?

Reply:
oops. that was temporary for testing purpose. :D I'll revert it.